UnrealPlugin/TestingProject/Plugins/PlayFabGSDK/Source/PlayFabGSDK/PlayFabGSDK.Build.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Unfortunately there's no way I've found to reference external code, so the copy is necessary. The way I ended up working was just modifying the plugin in the project to test effectively then copying to the sdk source when I was ready to commit. I briefly thought about what a script may be able to do facilitate the workflow, but it would only really be a single copy command and not incredibly useful. I'm considering a pipeline step to check the sdk source and the plugin in the test project are identical though.